CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Self-reference models - Users can update their record if there're a belongsTo and a hasMany self-reference associations.
```

Comment:
Self-referenced models - Users can update their records if the model has both a belongsTo and a hasMany self-reference associations.